setup.py

```diff
@@ -41,8 +42,10 @@
             "black==20.8b1",
             "flake8==3.9.0",
             "flake8-import-order==0.18.1",
+            "mypy==0.982",
             "pytest==6.2.2",
             "pytest-cov==2.11.1",
+            "types-requests"
```

Comment:
Freeze version if possible

Reply:
sure, I froze it the the same version as requests